scripts/setup_web1.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# CAUTION: To git clone olsystem, environment variables must be set...
+# Set $GITHUB_USERNAME or $USER will be used.
+# Set $GITHUB_TOKEN or this script will halt.
+if [[ -z ${GITHUB_TOKEN} ]]; then
+    echo "FATAL: Can not git clone olsystem" ;
+    exit 1 ;
+fi
+
+# apt list --installed
+sudo apt-get update
+sudo apt-get install -y docker.io docker-compose
+docker --version        # 19.03.8
+docker-compose version  #  1.25.0
+sudo systemctl start docker
+sudo systemctl enable docker
+
+sudo groupadd --system openlibrary
+sudo useradd --no-log-init --system --gid openlibrary --create-home openlibrary
+
+cd /opt
+ls -l  # nothing
+
+# sudo git clone https://github.com/internetarchive/openlibrary
+sudo git clone https://github.com/cclauss/openlibrary
+sudo git clone https://${GITHUB_USERNAME:-$USER}:${GITHUB_TOKEN}@github.com/internetarchive/olsystem
+# sudo git pull https://${GITHUB_USERNAME:-$USER}:${GITHUB_TOKEN}@github.com/internetarchive/olsystem.git master
```

Comment:
What is the way to clone `olsystem` that does not require passing in a token?

Reply:
It may be worth looking into how it's done for kubernetes -- I think we can assume that olsystem will be manually pulled / on the provisioned VM (from ansible) or that a key can be provided.
 https://git.archive.org/www/openlibrary/-/blob/master/Dockerfile#L11-18

Let's work w/ @abezella and make this as easy as possible on ourself. Docker doesn't have to magic all the things